docs/.markdownlint-cli2.cjs

```diff
@@ -1,164 +0,0 @@
-/*!
```

Comment:
TODO: replace with [remark-link](https://github.com/remarkjs/remark-lint) for better synergy with the rest of the Docusaurus build tooling